.env-copy

```diff
@@ -0,0 +1,3 @@
+# These are used for testing purposes
+RPC_HTTP_HOST=https://node.rugpullindex.com
```

Comment:
In the strategies repository we can just assume that neume-network/core always delivers the respective env vars. In the tests, while we can't expect it: we should be conscious when writing test cases. I recommend removing this file to not create the wrong impression.

Alternatively, env vars can be defined in e.g. package.json "ava" or GitHub actions